### DISCUSSION TITLE
Add RTSP stream to Frigate? (or help me figure out how to get frigate to work with Roku)

### DISCUSSION THREAD

**Discussion by ryddler** [blakeblackshear/frigate#498] [Ask A Question]
Prior to my installation of frigate I was using motioneye, which gave me the ability to stream to the IP Camera Viewer channel for my Roku. With frigate I haven't been able to do that any more, so I'm just wondering if anyone else has used this channel, whether you were able to get it working, or if RTSP in frigate is a viable option.

**Comment by mattheys** [comment]
That skill says it only takes RTSP in jpeg only?! I use http://github.com/aler9/rtsp-simple-server as a proxy to reduce network bandwidth by allowing you to run multiple services and apps against a camera whilst only pulling from the camera once which might help you. Doesn't transcode though so not sure if it will help as most cameras now I think are h264/h265 and not jpeg.

**Comment by paularmstrong** [comment]
Are you looking at [this app](https://channelstore.roku.com/details/4d75a841b5ca64054d0d577cd68aa043/ip-camera-viewer-pro)?

> Motion-JPEG streams, RTSP (jpg encoded)

This is `mjpeg`, which is what the `detect` camera role provides and you can access it from `http://your-ip:5000/<camera_name>` ([docs](https://github.com/blakeblackshear/frigate#camera_name), [beta docs](https://github.com/blakeblackshear/frigate/tree/release-0.8.0#camera_name))

**Reply by ryddler** [reply]
Yes, that's the app, but I haven't been able to get it to work. I put in the ip, the port, and set up /<camera_name> in the app, but it never comes up with the stream
